src/styles/blocks/card.scss

```
@@ -0,0 +1,93 @@
+.card {
+  font-family: $main-font;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &__image {
+    background-image: url("../images/imac.jpeg");
+    width: 160px;
+    height: 134px;
+    background-size: 160px 134px;
+    margin: 32px 19px 40px;
+    align-self: center;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $primary-title-color;
+  }
+  
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $primary-grey-color;
+    margin-bottom: 16px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 23px;
+  }
+
+  &__feedback {
+    font-family: $main-font;
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+    color: $primary-title-color;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+  }
+
+  &__price-text {
+    font-size: 12px;
+    font-weight: 400;
+    color: $primary-grey-color;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    font-weight: 700;
+    color: $primary-title-color;
+  }
+
+  &__link {
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    color: $primary-white-color;
+    width: 166px;
+    height: 40px;
+    margin: 0 auto;
+    background-color: $accent-blue-color;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-transform: uppercase;
+    text-decoration: none;
+    border-radius: 5px;
+  }
+
+  &__link:hover {
+    color: $accent-blue-color;
+    background-color: $primary-white-color;
+    border: 1px solid $accent-blue-color;
```

Don't add new border to the element on hover. Add default transparent border of the same width, and change its color on :hover
```suggestion
    border-color: $accent-blue-color;
```